NOTICE

```diff
@@ -1,5 +1,5 @@
-mesher
-Copyright 2018-present, Huawei
+Apache ServiceComb Mersher
+Copyright 2017-2019 The Apache Software Foundation
```

Comment:
We should keep the old copyright (Copyright 2018-2019, Huawei) and new Apache one (Copyright 2019-present, The Apache Software Foundation)